### PR TITLE
Clean up err codes

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0136.md [hunk 1]
```diff
@@ -14,3 +13,7 @@ fn main() { // error!
     // ...
 }
 ```
+
+A binary can only have one entry point, and by default that entry point is the
+`main()` function. If there are multiple instances of this function, please
+rename one.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0136.md]:
```suggestion
rename one of them.
```

File: src/librustc_error_codes/error_codes/E0161.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-A value was moved. However, its size was not known at compile time, and only
-values of a known size can be moved.
+A value was moved while size was not known at compile time.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0161.md]:
```suggestion
A value was moved whose size was not known at compile time.
```

File: src/librustc_error_codes/error_codes/E0164.md [hunk 3]
```diff
@@ -1,24 +1,42 @@
-This error means that an attempt was made to match a struct type enum
-variant as a non-struct type:
+Something which isn't a tuple struct nor a tuple variant was used as a pattern.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0164.md]:
```suggestion
Something which is neither a tuple struct nor a tuple variant was used as a pattern.
```

File: src/librustc_error_codes/error_codes/E0164.md [hunk 4]
```diff
@@ -1,24 +1,42 @@
...
     match foo {
-        Foo::B(i) => i, // error E0164
+        A::new() => (), // error!
+        _ => {}
     }
 }
 ```
 
-Try using `{}` instead:
+This error means that an attempt was made to match something which isn't a tuple
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0164.md]:
```suggestion
This error means that an attempt was made to match something which is neither
```

File: src/librustc_error_codes/error_codes/E0164.md [hunk 5]
```diff
@@ -1,24 +1,42 @@
...
+struct nor a tuple variant. Only those two elements are allowed as pattern:
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0164.md]:
```suggestion
a tuple struct nor a tuple variant. Only these two elements are allowed as pattern:
```

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

**Comment by GuillaumeGomez** [comment]
Updated! Thanks for the review @Dylan-DPC ! :)

**Pull request merged**